lsp/README.md

```diff
@@ -18,6 +18,14 @@ To enter a shell with `nickel` and `nls` run
 nix shell github:tweag/nickel
 ```
 
+Alternatively to install both using nix run:
+
+```
+git clone https://github.com/tweag/nickel.git
+cd nickel
+nix-env -f . -i
```

Comment:
Should we rather endorse the new nix command `nix build` or mention that this is the method to install without flakes?

Reply:
I have no preference.

Reply:
We should at least mention `nix build` or `nix profile install` as they are turning into the standard commands for nix

Reply:
Added